reference/tls/cleartext-redirection.md

```diff
@@ -21,7 +21,35 @@ Ambassador Edge Stack exposes configuration for this in two ways:
 
 Typically, port-based redirection is the preferred method since it is simpler to manage and will work with all use cases. Redirecting based off of the `x-forwarded-proto` header requires an L7 load-balancer or proxy in front of Ambassador Edge Stack to set that header.
 
-## Port-based Redirection
+## Using the `Host` resource
+
+The [`Host`](/reference/host-crd) resource is used to configure how Ambassador handles cleartext on a domain. You can configure Ambassador to [deny, allow, or redirect](/reference/host-crd/#secure-and-insecure-requests) cleartext to HTTPS on a domain using a single resource.
+
+The following `Host` gives an example of how to enable Ambassador to redirect cleartext to HTTPS on the `host.example.com` domain. 
+
+```yaml
+---
+apiVersion: getambassador.io/v2
+kind: Host
+metadata:
+  name: example-host
+spec:
+  hostname: host.example.com
+  acmeProvider:
+    email: julian@example.com
+  insecure:
+    action: Redirect
+```
+
+**Note:** Ambassador defaults to redirecting cleartext to HTTPS for all `Host`s. 
+
+## For Ambassador 0.86.1 and below
+
+In Ambassador 0.86.1 and below, TLS behavior was configured with the `TLSContext`. This included cleartext redirection. 
+
+The follow reference is **deprecated**. For those using old versions of Ambassador, [upgrade to the latest verion](/user-guide/upgrade-to-edge-stack/) of Ambassador to use the `Host` resource to manage TLS termination. 
```

Comment:
Does this section encompass port and protocol based redirection? If so, we should retitle this h2 to be "TLSContext (Deprecated)" and then open with the versioning info. 

Then I would say "The following sections for TLSContext, protocol and port based redirection, are also deprecated."

Reply:
Additionally, if .86 is only a version/applicable for the API Gateway, we should remove any AES wording here.

Reply:
Yes it does which is why `port-based redirection` and `protocol-based redirection` are both h3.

`TLSContext (Deprecated)` is not really correct since the `protocol-based redirection` is not configured with the `TLContext`